client/client_test.go

```diff
@@ -7564,7 +7565,7 @@ func testExportAttestations(t *testing.T, sb integration.Sandbox) {
 
 			purls := map[string]string{}
 			for _, k := range targets {
-				p, _ := purl.RefToPURL(k, &ps[i])
+				p, _ := purl.RefToPURL(packageurl.TypeDocker, k, &ps[i])
```

Comment:
it would be better if this used string constants instead. Otherwise we are running the same function both in implementation and in test.

So did these values actually change as well?

Reply:
The values haven't changed, no, we just move the `packageurl.TypeDocker` from inside the function to outside the function.

I've changed these to string constants (still need a bit of printf magic) - I've put this commit at the start of the patch stack, and verified that the values stay constant with each patch.